apps/native-component-list/web/index.html

```diff
@@ -29,6 +29,12 @@
     <meta name="msapplication-TileColor" content="" />
     <meta name="msapplication-TileImage" content="" />
 
+    <script
```

Comment:
I think we should we:
1. Add docs about writing this yourself
2. Add an optional utility to inject this script from JS (not a great idea)